.github/workflows/ci.yaml

```diff
@@ -79,12 +79,9 @@ jobs:
           key: ${{ runner.os }}-vscode-1.70.2
 
       - name: Run E2E tests
-        uses: GabrielBB/xvfb-action@v1.0
+        uses: coactions/setup-xvfb@v1.0
         with:
           run: npm run test:e2e
-        env:
-          CODE_VERSION: ${{ matrix.version }}
```

Comment:
`CODE_VERSION: ${{ matrix.version }}` was removed since we don't have a `version` property in this job's `matrix`.